assets/sass/cds/_components.scss

```diff
@@ -165,12 +170,17 @@
 		margin-bottom: 2rem;
 	}
 
+
+
 	.team-member {
 		position: relative;
-		border-bottom: $primary-color 4px solid;
+		border-bottom: $team-purple 4px solid;
 		box-shadow: rgb(225, 228, 231) 0px -1px 4px;
+		background-color: $medium-grey;
 		padding: 2.5rem 0 0;
 		margin-bottom: 2rem;
+		height: 482px;
```

Comment:
we might want to bump this height down a bit for mobile only, but I leave that up to others